src/shogun/multiclass/tree/CARTree.cpp

```diff
@@ -1452,6 +1524,9 @@ void CARTree::init()
 	m_sorted_features=SGMatrix<float64_t>();
 	m_sorted_indices=SGMatrix<index_t>();
 
+	SG_ADD(
+	    &m_feature_importances, "feature_importances", "feature importances");
```

Comment:
Could you add .get("feature_importances") in cartree.sg? So we can see how it works in interfaces

Reply:
done :)